### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by Benawi**
### Features Added 
- jest framework Used for testing.
- *describe()* method is uded to group test cases
- A test file (add-remove.test.js) [L1C1-L72](https://github.com/Benawi/Microverse-TestingToDolist/blob/99d01b6e450c1bc24be68f746ecbd8071540f510/src/test/update.test.js#L1C1-L72) is created and containing the following functions:
   - A function for editing the task description.
   - A function for updating an item's 'completed' status.
   - The "Clear all completed" function.
   - The mock storage object usedwhich been  have created in Part 1 of this project to mock the localStorage updates.

### CODE REVIEW

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Benawi, @Peter-Dumbari   , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights

- Tests are passing✔✔
- Descriptive pull request summary :heavy_check_mark: 

 ## Required Changes ♻️
- [ ] You did a great job, :+1:  :+1:  All your tests are passing.. But as required by Microverse, all links on the readme must be working, Current your LICENSE link is currently broken, it leads to a 404 error page, I'll suggest you add an MIT license to your project and link it correctly. :smile: :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
### Changes ♻️
-  The broken LICENSE link is fixed

File: README.md [hunk 1]
```diff
@@ -22,34 +22,33 @@
 
 - [📝 License](#license)
 
-
 # 📖 Testing Project <a name="about-project"></a>
 
 Testing Microverse! project is a repository consisting of the following files:
+
```
Comment (tufoinnkuo10) [inline README.md:28]: - [optional] it's better to remove the live demo link section now to keep your project README file clean. You can add it later on when your project is deployed.
- [optional] please the future features section should include features or extra ones you will add to your application in future or time to come, not a question asked. Please look into that.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Team

Your project is complete! Great job making the requested change 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Test ok
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
### Changes ♻️
Hi @tufoinnkuo10 
- [optional]  comments fixed.

**Pull request merged**